scripts/metrics_to_tensorboard.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# Add metrics images to tensorboard summary for easy viewing/comparisons
+
+import argparse
+import os
+
+from PIL import Image
```

Comment:
Does `debug_run.py` fail if either PIL or torchvision are not installed? I'm assuming it just ends up swallowing the ImportError if invoked by debug_run.py?

Reply:
I have not tested it without those two libraries. In the PyTorch images that we release, both of these libraries are installed. Let me test it.

Reply:
I introduced an artificial import error by changing the module name from ToTensor to ToTensor**s**.  It seems to report the trace: ```
Traceback (most recent call last):
  File "/home/sivaibhav/xla/scripts/metrics_to_tensorboard.py", line 11, in <module>
    from torchvision.transforms import ToTensors
```

Reply:
Erm given that most users will have PIL preinstalled and torchvision, I think it's reasonable, since it doesn't stop debug_run.py.